web/src/stories/table.stories.mdx

```diff
@@ -11,6 +11,18 @@ table.Add(component.TableRow{
 table.Sort("A")
 table.Reverse()`}}
 
+export const tableExpandableRowDocs= {source: {code: `cols := component.NewTableCols("A", "B")
```

Comment:
This story needs more work: 
- please add more details about this feature to docs section, 
- the size of each section can be uniformly reduced,
- section headers need to be added,
- there should be only one Props section at the bottom,

See for example the docs page for Dropdown component

Reply:
There has to be two props since `Table` can mean two different components. The remaining sections are updated to match the Dropdown section as much as possible.

Reply:
Looks much better now, thanks! Very small detail: the docs section would look better if height of 1st table is ~250px and 2nd ~350.